.github/workflows/arduino.yml

```diff
@@ -17,12 +17,16 @@ jobs:
   build:
     # The type of runner that the job will run on
     runs-on: windows-latest
+    strategy:
```

Comment:
I was under the impression that this workflow was conditioned to only run on PRs that touched the relevant directories (Arduino, or ArduinoCsCompiler). Can we change it so that is the case?

Reply:
That is handled within the build - when those parts have not changed, only a limited number of tests are run. Only if they did change, the full test suite runs.